deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -687,3 +779,16 @@ def infrasetup_instance(self) -> None:
     def terminate_instance(self) -> None:
         """ VitisInstanceDeployManager machines cannot be terminated. """
         return
+
+class AzureInstanceDeployManager(VitisInstanceDeployManager):
```

Comment:
IIRC, I think you plan to remove the azure support before the final merge right?

Reply:
These are necessary but not sufficient changes to add Azure support. I don't have access to running an `.xclbin` flow anywhere except for Azure. So I was thinking of leaving these in for now.